murdock/murdock.py

```diff
@@ -210,13 +214,17 @@ async def add_job_to_queue(self, job: MurdockJob):
                 "target_url": self.base_url,
             },
         )
-        all_busy = all(running is not None for running in self.running.jobs)
         self.queued.add(job)
         job.state = "queued"
-        if all_busy and job.fasttracked:
-            self.fasttrack_queue.put_nowait(job)
+        if job.fasttracked:
+            prio = JOB_FASTTRACK_BONUS
         else:
-            self.queue.put_nowait(job)
+            prio = 0
+
+        # PriorityQueue is a min_queue, but "higher priority equals higher value"
+        # makes more sense. So, invert the priority by multiplying with `-1`
+        self.queue.put_nowait(PrioritizedItem(prio * -1, job))
```

Comment:
I suggest wrapping this into a separate function. The `prio * -1` makes this error prone when copy-pasted around if used elsewhere.

```Python
def _submit(self, job: Any, prio: int):
    """
    Submit a job into the priority queue. Jobs with a high prio value are served before jobs with low prio value.

    :param job: Job to submit
    :param prio: Priority of the job
    """
    self.queue.put_nowait(PrioritizedItem(prio * -1, job))
```

Reply:
would that be a function level function? or class level?

Reply:
@aabadie Any opinion here?

Reply:
If the priority is added as an attribute to MurdockJob, there won't be a need for wrapper item. The `__le__` method of MurdockJob could take care of inverting the order.